ui/raidboss/data/07-dt/deepdungeon/pilgrims_traverse_stones_81-90.ts

```diff
@@ -0,0 +1,246 @@
+      type: 'StartsUsing',
+      netRegex: { id: 'AEF0', source: 'Invoked Cerberus', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid AoE',
+        },
+      },
+    },
+    {
+      id: 'PT 81-90 Invoked Cerberus Hellclaw',
+      type: 'StartsUsing',
+      netRegex: { id: 'AEF1', source: 'Invoked Cerberus', capture: false },
+      response: Responses.awayFromFront(),
+    },
+    {
+      id: 'PT 81-90 Invoked Cerberus Tail Blow',
+      type: 'StartsUsing',
+      netRegex: { id: 'AEF2', source: 'Invoked Cerberus', capture: false },
+      response: Responses.goFront(),
+    },
+    {
+      id: 'PT 81-90 Invoked Humbaba Triple/Quadruple Blow',
+      type: 'StartsUsing',
+      netRegex: { id: ['A93B', 'A93C'], source: 'Invoked Humbaba', capture: true },
+      infoText: (_data, matches, output) => {
+        const blows = matches.id === 'A93B' ? 3 : 4;
+        return output.text!({ count: blows });
+      },
+      outputStrings: {
+        text: {
+          en: '${count}x attacks => Get Behind',
+        },
+      },
+    },
+    {
+      id: 'PT 81-90 Invoked Humbaba Bellows',
+      type: 'StartsUsing',
+      netRegex: { id: 'AD05', source: 'Invoked Humbaba', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'PT 81-90 Invoked Caym Double Hex Eye',
+      type: 'StartsUsing',
+      netRegex: { id: 'AEEE', source: 'Invoked Caym', capture: true },
+      response: Responses.lookAwayFromSource(),
+    },
+    {
+      id: 'PT 81-90 Invoked Baal Incinerating Lahar',
+      type: 'StartsUsing',
+      netRegex: { id: 'A87D', source: 'Invoked Baal', capture: true },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 4,
+      alertText: (_data, matches, output) => output.breakLOS!({ name: matches.source }),
+      outputStrings: {
+        breakLOS: {
+          en: 'Break line-of-sight to ${name}',
+          de: 'Unterbreche Sichtlinie zu ${name}',
+          fr: 'Masquez le champ de vision vers ${name}',
+          ja: '${name}の視線から隠れる',
+          cn: '利用掩体卡 ${name} 的视线',
+          ko: '${name}의 시야 밖으로 숨기',
+        },
+      },
+    },
+    {
+      id: 'PT 81-90 Invoked Baal Abyssal Ray',
+      // goes through walls
+      type: 'StartsUsing',
+      netRegex: { id: 'A87E', source: 'Invoked Baal', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'PT 81-90 Traverse Cama Claw and Tail',
+      type: 'StartsUsing',
+      netRegex: { id: 'A87C', source: 'Traverse Cama', capture: false },
+      response: Responses.goSides(),
+    },
+    // ---------------- Stone 90 Boss: Malacoda ----------------
+    {
+      id: 'PT 81-90 Malacoda Backhand Right',
+      type: 'StartsUsing',
+      netRegex: { id: 'ACDA', source: 'Malacoda', capture: false },
+      response: Responses.goLeft(),
```

Comment:
For the Backhand outputs, is it worth customizing a response for it to be "behind + left/right"?